src/commands/force/data/bulk/delete.ts

```diff
@@ -41,13 +42,13 @@ export default class Delete extends DataCommand {
     let result: BulkResult[] | JobInfo[];
 
     try {
-      await this.throwIfFileDoesntExist(this.flags.csvfile as string);
+      await this.throwIfPathDoesntExist(this.flags.csvfile as string);
 
       const conn: Connection = this.ensureOrg().getConnection();
       this.ux.startSpinner('Bulk Delete');
 
       const csvRecords: ReadStream = fs.createReadStream(this.flags.csvfile as string, { encoding: 'utf-8' });
-      const job: Job = conn.bulk.createJob(this.flags.sobjecttype as string, 'delete');
+      const job: Job = conn.bulk.createJob(this.flags.sobjecttype, 'delete') as unknown as Job;
```

Comment:
Can you explain the typescript change here?

Reply:
Could we do the same thing [here](https://github.com/salesforcecli/plugin-data/pull/284/files#diff-256d4db6127a97e7087811a8620c74cc01748340e4cc1e6cf9da0ef0aec6c88aR61) instead of doing ts-ignore?

Reply:
The `jsforce` v2 `Job` had a few minor changes which casts to `Job<Schema, operation>`, but we just need to know it's a `Job`

probably - I'll look